docs/scrnaseq/README.md

```diff
@@ -209,6 +210,7 @@ The method by which specific cell populations are sorted or enriched. Leave blan
 | constraint | value |
 | --- | --- |
 | required | `False` |
```

Comment:
@cebriggs7135 : Does it still make sense for this field to be optional, or should they just supply the string `none`, to be consistent with atacseq?

Reply:
Might as well keep it consistent with atacseq by allowing "none".